.travis.yml

```diff
@@ -1,53 +1,53 @@
-sudo: false
-
-cache:
-  apt: true
-  directories:
-  - $HOME/.cache/pip
-  - $HOME/.ccache
-  - $HOME/.pip-cache
-dist: trusty
+branches:
+  only:
+    - develop
```

Comment:
What does this file actually do? It looks to me like it will build a wheel on every commit, which isn't what we want, right?

Reply:
I guess not. I can build on every release. Is that done through master?

Reply:
Releases are done on each tag that matches the regex `v\d+\.\d+\.\d+`

Reply:
You can create a cron job from the UI. Can't find any documentation about setting it up through the .travis.yml file. https://docs.travis-ci.com/user/cron-jobs/